sunpy/net/hek/coord_properties.json

```diff
@@ -0,0 +1,564 @@
+            "type": "float",
+            "AR": "req",
+            "CE": "req",
+            "CD": "req",
+            "CH": "req",
+            "CW": "req",
+            "FI": "req",
+            "FE": "req",
+            "FA": "req",
+            "FL": "req",
+            "LP": "req",
+            "OS": "req",
+            "SS": "req",
+            "EF": "req",
+            "CJ": "req",
+            "PG": "req",
+            "OT": "req",
+            "NR": "req",
+            "SG": "req",
+            "SP": "req",
+            "CR": "req",
+            "CC": "req",
+            "ER": "req",
+            "TO": "req",
+            "HY": "req",
+            "BU": "req",
+            "EE": "req",
+            "PB": "req",
+            "PT": "req",
+            "desc": "Coord2 of upper-right corner of bounding box",
+            "is_unit_prop": false,
+            "is_chaincode": false,
+            "unit_prop": "coord2_unit"
+        },
+        {
+            "name": "skel_chaincode",
+            "type": "string",
+            "CH": "opt",
+            "FI": "opt",
+            "FE": "opt",
+            "FA": "opt",
+            "FL": "opt",
+            "LP": "req",
+            "CJ": "opt",
+            "PG": "opt",
+            "OT": "opt",
+            "NR": "opt",
+            "SG": "opt",
+            "SP": "opt",
+            "CR": "opt",
+            "CC": "opt",
+            "ER": "opt",
+            "TO": "opt",
+            "desc": "Skeleton code. A skeleton code is like a chain code except it isn't closed. (ordered list delimited by commas. e.g. 'x1, y1, x2, y2, x3, y3')",
+            "is_unit_prop": false,
+            "is_chaincode": true
+        },
+        {
+            "name": "skel_startc1",
+            "type": "float",
+            "CH": "opt",
+            "FI": "opt",
+            "FE": "opt",
+            "FA": "opt",
+            "FL": "opt",
+            "LP": "req",
+            "CJ": "opt",
+            "PG": "opt",
+            "OT": "opt",
+            "NR": "opt",
+            "SG": "opt",
+            "SP": "opt",
+            "CR": "opt",
+            "CC": "opt",
+            "ER": "opt",
+            "TO": "opt",
+            "desc": "Beginning Coord 1 of skeleton",
+            "is_unit_prop": false,
+            "is_chaincode": false,
+            "unit_prop": "coord1_unit"
+        },
+        {
+            "name": "skel_startc2",
+            "type": "float",
+            "CH": "opt",
+            "FI": "opt",
+            "FE": "opt",
+            "FA": "opt",
+            "FL": "opt",
+            "LP": "req",
+            "CJ": "opt",
+            "PG": "opt",
+            "OT": "opt",
+            "NR": "opt",
+            "SG": "opt",
+            "SP": "opt",
+            "CR": "opt",
+            "CC": "opt",
+            "ER": "opt",
+            "TO": "opt",
+            "desc": "Beginning Coord 2 of skeleton",
+            "is_unit_prop": false,
+            "is_chaincode": false,
```

Comment:
Do we need the false values?

Reply:
Removed them and refactored the code for these changes